tensorboard/pip_package/test_pip_package.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/sh
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
2019

Reply:
Sure. (I went with 2015 because it’s a file split, but 2019 is fine.)